src/backend/utils.py

```diff
@@ -140,14 +140,13 @@ def get_image_names_in_a_folder(relative_folder_path):
     """
 
     data_dir = get_data_directory()
-    path = str(Path(data_dir + relative_folder_path)) + '\*.jpg'
+    path = str(Path(data_dir + relative_folder_path)) + '/*.jpg'
```

Comment:
This change is because of ubuntu?? or are you also using windows?

Reply:
Yes, on Windows both \ and / work but on Ubuntu only / works.